apiary.apib

```diff
@@ -3315,6 +3317,151 @@ Get information about a meeting.
+                + type: `company` (string)
+            + milestone_id: `f29abf48-337d-44b4-aad4-585f5277a456` (string, optional) - Filter calls by milestone
+            + call_outcome_id: `6bd20dd8-dc3f-0e68-8919-dee0be6efe55` (string, optional) - Filter on done calls by outcome
+        + page (object, optional)
+            + size: `20` (number, optional) - The amount of entries returned per request, max 1 to 100
+                + Default: `20`
+            + number: `1` (number, optional) - The current page, the first page is 1
+                + Default: `1`
+
++ Response 200 (application/json)
+
+    + Attributes (object)
+        + data (array)
+            + (object)
+                + id: `32665afd-1818-0ed3-9e18-a603a3a21b95` (string)
+                + added_at: `2021-11-21T18:14:15+00:00` (string, nullable)
+                + customer (object, nullable)
+                    + id: `32665afd-1818-0ed3-9e18-a603a3a21b95` (string)
+                    + type: `company` (string)
+                + description: `Call with customer` (string, nullable)
+                + optional_contact (object, nullable)
+                    + id: `32665afd-1818-0ed3-9e18-a603a3a21b95` (string)
+                    + type: `contact` (string)
+                + outcome (object, nullable)
+                    + type: `callOutcome` (string)
+                    + id: `32665afd-1818-0ed3-9e18-a603a3a21b95` (string)
+                + outcome_summary: `Called, but was not available` (string, nullable)
+                + responsible_user (object, nullable)
+                    + id: `f29abf48-337d-44b4-aad4-585f5277a456` (string)
+                    + type: `user` (string)
+                + scheduled_at: `2021-11-21T19:14:15+00:00` (string)
+                + status: `open` (enum[string])
+                    + Members
+                        + `open`
+                        + `done`
+        + meta (object) - Only included with request parameter `includes=pagination`
+            + page (Pagination)
+            + matches: `12` (number)
+
+### calls.info [POST /calls.info]
+
+Get information about a call.
+
++ Request (application/json)
+
+    + Attributes (object)
+        + id: `6fac0bf0-e803-424e-af67-76863a3d7d16` (string, required)
+
++ Response 200 (application/json)
+
+    + Attributes (object)
+        + data (object)
+            + id: `32665afd-1818-0ed3-9e18-a603a3a21b95` (string)
+            + added_at: `2016-02-01T16:44:33+00:00` (string, nullable)
+            + completed_at: `2016-02-04T16:44:33+00:00` (string, nullable)
+            + customer (object, nullable)
+                + type: `company` (string)
+                + id: `2659dc4d-444b-4ced-b51c-b87591f604d7` (string)
+            + description: `Description of the call` (string, nullable)
+            + optional_contact (object, nullable)
+               + type: `contact` (string)
+               + id: `2659dc4d-444b-4ced-b51c-b87591f604d7` (string)
```

Comment:
Q: Can I assume that the optional contact is part of the "customer" you are calling?

Reasoning, I call a company where I have a contact, which I assume `optional_contact` is for?
From what I see this contact when added to the call is automatically linked to the company the call is for too, so this would make sense to have it structured like 👇 

They should probably be grouped:
```suggestion
            + recipient (object, nullable)
               + customer: (object, required)
                  + type: `company` (string)
                  + id: `2659dc4d-444b-4ced-b51c-b87591f604d7` (string)
               + for_attention_of: (object, nullable)
                  + type: `contact` (string)
                  + id: `2659dc4d-444b-4ced-b51c-b87591f604d7` (string)
            + description: `Description of the call` (string, nullable)
```

Reply:
As an addition, if we look at a similar structure on `invoices.list`, should we group the 'type' and 'id' of `for_attention_of` in a `contact` object? This would open up the options to add expose additional properties in the future.

```suggestion
            +recipient (object, nullable)
               + customer: (object, required)
                  + type: `company` (string)
                  + id: `2659dc4d-444b-4ced-b51c-b87591f604d7` (string)
               + for_attention_of: (object, nullable)
               	  + contact (object, required)
                    + type: `contact` (string)
                    + id: `2659dc4d-444b-4ced-b51c-b87591f604d7` (string)
            + description: `Description of the call` (string, nullable)
```